src/test/java/calculator/app/CalculatorTest.java

```diff
@@ -0,0 +1,92 @@
+package calculator.app;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class CalculatorTest {
+
+    private static Calculator calculator;
+
+    @BeforeAll
+    static void setMain() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    @DisplayName("Test addition functionality")
+    void givenTwoNumbers_whenAdditionFunctionality_thenCorrectResult() {
+        //given
+        Integer a = 10;
+        Integer b = 5;
+        Integer expected = 15;
+
+        //when
+        Integer actual = calculator.addition(a, b);
+
+        //then
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @DisplayName("Test subtraction functionality")
+    void givenTwoNumbers_whenSubtractionFunctionality_thenCorrectResult() {
+        //given
+        Integer a = 10;
+        Integer b = 5;
+        Integer expected = 5;
+
+        //when
+        Integer actual = calculator.subtraction(a, b);
```

Comment:
one test for 1 method is not enough in this case, because we need to test differenet scenarios.

Always keep in mind, that unit tests and testing at all, should not be performed to prove that your program is working.

You should write tests like you want to find, that your code is NOT working.
In other words, imagine, someone propose you 100 000 dollars and say "Just find scenario, that will make your code fail".

It's different mindset, otherwise tests will be just bad and for checkbox.

In this case, you could use corner / edge case.
2 + 5 work, ok, what about 5 + 0? Zero is special number

Then what about negative numbers, -100 + -40 or -100 + 40

Then what about large number - 34930294 + 32538394

In business logic usually it's enough to cover conditions (IF ELSE branching) or HappyPath / sunnyday scenario when all is ok, and negative if it could be exception.

In this case we deal with numbers so we need to think about different possible scenarios.

To make easier and faster, I recommend use Parameterize testing, just use CsvSource or MethodSource and 3-4 scenario for each method (default, with zero, negative, large numbers)